### DISCUSSION TITLE
Uncluster Marker Pin

### DISCUSSION THREAD

**Discussion by Niko-La** [opengeos/leafmap#98] [Q&A] [answered]
Map.add_xy_data(in_csv_5, x="Longitude", y="Latitude", layer_name="World")

Is there a way to change the settings in leafmap to not cluster pins when zoomed put?

also is there a way to change the color of marker pins so you can a few different colors to show different locations (school in green pin, hosptial in blue pin, firestation in red pin)?

**Comment by giswqs** [comment, marked as answer]
> Is there a way to change the settings in leafmap to not cluster pins when zoomed put?

Unfortunately, the answer is No. Maker Cluster is not customizable except for the marker locations. See https://ipyleaflet.readthedocs.io/en/latest/api_reference/marker_cluster.html. If you don't want the clusters when zoomed out, you will need to add each location as an individual marker/layer. Adding a few individual markers is fine, but it could be overwhelming to add hundreds of markers/layers. See https://github.com/jupyter-widgets/ipyleaflet/issues/498

> also is there a way to change the color of marker pins so you can a few different colors to show different locations (school in green pin, hosptial in blue pin, firestation in red pin)?

Check out AwesomeIcon 
https://ipyleaflet.readthedocs.io/en/latest/api_reference/awesome_icon.html
